### PR TITLE
Anne-edit-introduction

### PR DESCRIPTION
**Pull request by theredgeek**
A few styling changes in some descriptions changed the variables' names to be consistent and reflect the meaning of the code.

### CODE REVIEW

**Comment by PabRod** [comment]
Some context for @jhidding: 

@theredgeek discussed with me some ideas about this in a meeting yesterday. I agreed, so consider my review as approved :)

**Review by jhidding — CHANGES_REQUESTED** [review]
The code changes are all good. The phrasing of some of the sentences could be better. Overall the text in this section is a bit too wordy for my taste. We should think about the target audience here: teachers of the course. The allegory of working people is a nice example that helps a teacher, but we shouldn't overdo it.

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -89,7 +87,7 @@ Suppose we have a computation where each step **depends** on a previous one. We
 
 In these diagrams the inputs and outputs of each function are represented as rectangles. The inward and outward arrows indicate their flow. Note that the output of one function can become the input of another one. The diagram above is the typical diagram of a **serial computation**. If you ever used a loop to update a value, you used serial computation.
 
-If our computation involves **independent work** (that is, the results of the application of each function are independent of the results of the application of the rest), we can structure our computation like this:
+Our computation involves **independent work** if the results of the application of each of our functions are independent from the results of the application of the rest. In that instance we can structure our computation like this:
```
[the remark this answers is not in the document]
Comment (liekelotte) [reply _episodes/01-introduction.md:90]: Hey Johan, since Anne has left us, could you make the requested change and merge this PR? Thanks in advance :)

[the remark this answers is not in the document]
Comment (jhidding) [reply _episodes/01-introduction.md:90]: Komt in orde, ik ga hier morgen aan werken.
